raster/r.buffer/testsuite/test_buffer.py

```diff
@@ -0,0 +1,96 @@
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+from grass.gunittest.gmodules import SimpleModule
+import grass.script as gs
+
+
+class TestRBuffer(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.use_temp_region()
+        cls.runModule("g.region", raster="roadsmajor")
```

Comment:
I would make the region smaller, to make the test faster.
```suggestion
        cls.runModule("g.region", n=223000, s=220000, w=640000, e=643000, nsres=100)
```